.prettierignore

```diff
@@ -2,3 +2,6 @@
 
 /dist
 /coverage
+
+/.nx/cache
```

Comment:
Can this just be `/.nx`?

Reply:
I didn't add this, so I guess it was an automation I'd rather not touch.